blog/2022-08-26-blog.md

```diff
@@ -0,0 +1,23 @@
+#   **Wiedereinstieg in Python**
+
+Das heutige Thema war Python/Tigerjython was mir nicht wirklich gefällt, da das nur eine Bibliothek,
+die keiner im Alltag braucht und nutzlos ist.
```

Comment:
Das Thema war Turtlegrafik, welches eine Bibliothek ist... 
(Tigerjython ist eine in Java geschriebene Implementation von Python, welche zusätzliche Sprachkonstrukte wie `repeat` erlaubt. (Wird ausserhalb des Unterrichts aber nicht eingesetzt...))